lib/nerves_hub_link/configurator.ex

```diff
@@ -13,11 +13,26 @@ defmodule NervesHubLink.Configurator do
               remote_iex: false,
               socket: [],
               ssl: []
+
+    @type t() :: %__MODULE__{
+            device_api_host: String.t(),
+            device_api_port: String.t(),
+            device_api_sni: charlist(),
+            fwup_public_keys: [binary()],
+            fwup_devpath: String.t(),
```

Comment:
```suggestion
            fwup_devpath: Path.t(),
```

Reply:
Path.t() is `iodata` which i don't think would actually work

Reply:
i'm totally wrong. we just updated teh other lib to support this